Userland/Libraries/LibJS/Runtime/IteratorOperations.cpp

```diff
@@ -136,6 +152,18 @@ Completion iterator_close(Object& iterator, Completion completion)
     return completion;
 }
 
+// 7.4.6 IteratorClose ( iteratorRecord, completion ), https://tc39.es/ecma262/#sec-iteratorclose
+Completion iterator_close(Object& iterator, Completion completion)
+{
+    return iterator_close_(iterator, move(completion), IteratorHint::Sync);
+}
+
+// 7.4.8 AsyncIteratorClose ( iteratorRecord, completion ), https://tc39.es/ecma262/#sec-asynciteratorclose
```

Comment:
hmm, 7.4.8 already exists below - are the section numbers outdated?

Reply:
It seems so, I updated the wrong number in the same commit I added the new method. 
Or should these kind of things be done in a separate commit?

Reply:
Fine to do it in the same when you notice it while adding something new :)